packages/t/tailscale/package.yml

```
@@ -0,0 +1,25 @@
+name       : tailscale
+version    : 1.48.2
+release    : 1
+source     :
+    - https://github.com/tailscale/tailscale/archive/refs/tags/v1.48.2.tar.gz : 1c34c5c2c3b78e59ffb824b356418ff828653c62885b126d0d05f300218b36b5
+homepage   : https://tailscale.com
+license    : BSD-3-Clause
+component  : network.clients
+summary    : Private WireGuard® networks made easy
+description: |
+    The easiest, most secure way to use WireGuard and 2FA.
+networking : yes
+builddeps  :
+    - golang
+    - git
+setup      : |
+    %patch -p1 < $pkgfiles/defaults.patch
```

This should use `-i` for clarity:

```suggestion
    %patch -p1 -i $pkgfiles/defaults.patch
```